Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm

```diff
@@ -108,6 +109,8 @@
 
 #import "PDFKitSoftLink.h"
 
+extern "C" AXError _AXUIElementNotifyProcessSuspendStatus(AXSuspendStatus);
```

Comment:
Our modern practice is to put these into `SPI.h` headers, not just into the one file where they are used. Sprinkling them into different files leads to a mess over time, and we would like to avoid that.